script.module.pymysql/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pymysql"
+       name="pymysql"
+       version="0.8.0"
+       provider-name="smitchell6879">
```

Comment:
Author should be the original module author not the one packaging it for kodi

Reply:
Where do I change this at? Also the Project has 72 contributors so would the project name 'PyMySQL' work?

Reply:
https://pypi.python.org/pypi/PyMySQL

 Author: INADA Naoki